frontend/src/metabase-lib/v1/parameters/utils/targets.ts

```diff
@@ -147,29 +155,89 @@ export function buildTextTagTarget(tagName: string): ParameterTextTarget {
   return ["text-tag", tagName];
 }
 
-export function getParameterColumns(question: Question, parameter?: Parameter) {
+export function getParameterColumns(
+  question: Question,
+  parameter?: Parameter,
+): {
```

Comment:
It might be worth putting this in a named type?

Reply:
It won't be reused anywhere though. Maybe it's better to remove it and leave the work to TS in this case.
Removed: 7f2aa4b5f769d88ac01ee36a259cd0dc98837bdb